labshare/management/commands/update.py

```diff
@@ -40,7 +40,16 @@ def handle(self, *args, **options):
                         gpu.used_memory = memory_info["used"]
                         gpu.total_memory = memory_info["total"]
                         gpu.in_use = gpu_in_use
+
                     gpu.save()
+                    gpu.processes.all().delete()
+                    for process in gpu_data.get("processes", []):
+                        GPUProcess(
+                            gpu = gpu,
```

Comment:
please remove the spaces around the kwargs ;)
see https://www.python.org/dev/peps/pep-0008/#id28